frontend/src/pages/phone.page.tsx

```diff
@@ -83,13 +90,31 @@ const Phone: NextPage = () => {
   ) {
     return (
       <Layout runtimeData={runtimeData.data}>
-        <DashboardSwitcher />
-        <PhoneDashboard
-          profile={profile}
-          runtimeData={runtimeData.data}
-          realPhone={verifiedPhones[0]}
-          onRequestContactCard={() => realPhoneData.resendWelcomeSMS()}
-        />
+        <div>
+          <DashboardSwitcher />
+          {/* Only show the welcome screen if the user hasn't seen it before */}
+          {!welcomeScreenDismissal.isDismissed &&
+          isFlagActive(runtimeData.data, "multi-replies") ? (
```

Comment:
The flag name uses underscores: 
```suggestion
          isFlagActive(runtimeData.data, "multi_replies") ? (
```

Reply:
^ Good callout. My oversight, thanks.